opengrok-indexer/pom.xml

```diff
@@ -328,7 +328,6 @@ Portions Copyright (c) 2017-2018, Chris Fraire <cfraire@me.com>.
                             <lexDefinitions>
                                 <lexDefinition>${basedir}/target/jflex-sources</lexDefinition>
                             </lexDefinitions>
-                            <inputStreamCtor>false</inputStreamCtor>
```

Comment:
why not needed anymore?

Reply:
https://github.com/jflex-de/jflex/issues/195